<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="clientSearchTabWidget">
       <attribute name="title">
        <string>Recherche de client</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTableView" name="clientSearchTableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="staffListTabWidget">
       <attribute name="title">
        <string>Liste des ressources</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="StaffView" name="staffTreeView">
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="modifyStaffButton">
              <property name="text">
               <string>Modifier</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deleteStaffButton">
              <property name="text">
               <string>Supprimer</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="planificationTabWidget">
       <attribute name="title">
        <string>Planification</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>903</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuAdd">
     <property name="title">
      <string>Add</string>
     </property>
     <addaction name="actionCreateClient"/>
     <addaction name="actionCreateStaff"/>
    </widget>
    <addaction name="menuAdd"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionAboutUs"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionToolbarCreateClient"/>
   <addaction name="actionToolbarCreateStaff"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionToolbarCreateClient">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/user-plus.svg</normaloff>:/icons/user-plus.svg</iconset>
   </property>
   <property name="text">
    <string>Client</string>
   </property>
  </action>
  <action name="actionCreateStaff">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/address-card.svg</normaloff>:/icons/address-card.svg</iconset>
   </property>
   <property name="text">
    <string>Staff</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAboutUs">
   <property name="text">
    <string>About us</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+?</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionCreateClient">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/user-plus.svg</normaloff>:/icons/user-plus.svg</iconset>
   </property>
   <property name="text">
    <string>Client</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionToolbarCreateStaff">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/address-card.svg</normaloff>:/icons/address-card.svg</iconset>
   </property>
   <property name="text">
    <string>Staff</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>StaffView</class>
   <extends>QTreeView</extends>
   <header>views/staffview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
